app/lib/pdf_filler/nj1040_pdf.rb

```diff
@@ -137,6 +152,74 @@ def hash_for_pdf
         ]))
       end
 
+      if @xml_document.at("TotalIncome").present?
+        total_income = @xml_document.at("TotalIncome").text.to_i
+        answers.merge!(insert_digits_into_fields(total_income, [
+          "188",
+          "undefined_80",
+          "187",
+          "186",
+          "undefined_79",
+          "185",
+          "184",
+          "undefined_78",
+          "183",
+          "27",
+          "263"
+        ]))
+      end
+
+      if @xml_document.at("GrossIncome").present?
+        gross_income = @xml_document.at("GrossIncome").text.to_i
+        answers.merge!(insert_digits_into_fields(gross_income, [
+          "209",
+          "undefined_89",
+          "208",
+          "207",
+          "undefined_88",
+          "206",
+          "205",
+          "undefined_87",
+          "204",
+          "29",
+          "270",
+        ]))
+      end
+
+      if @xml_document.at("TaxableIncome").present?
+        gross_income = @xml_document.at("TaxableIncome").text.to_i
+        answers.merge!(insert_digits_into_fields(gross_income, [
+          "256",
+          "undefined_109",
+          "255",
+          "254",
+          "undefined_108",
+          "253",
+          "252",
+          "undefined_107",
+          "251",
+          "38a Total Property Taxes 18 of Rent Paid See instructions page 23 38a",
+          "279",
+        ]))
+      end
+
+      if @xml_document.at("NewJerseyTaxableIncome").present?
+        gross_income = @xml_document.at("NewJerseyTaxableIncome").text.to_i
```

Comment:
[Pebble] Similar question here: should this variable name be `nj_taxable_income`?